### DISCUSSION TITLE
Концепция лендинга "Индивидуальные занятия"

### DISCUSSION THREAD

**Discussion by Darolga** [torin-asakura/drum-in#29] [General]
@TorinAsakura
Готово три экрана. Макет [https://www.figma.com/file/7FTyeashb3rns6sCdrsz3b/Landing?node-id=51%3A1367](url)

На первом экране фоное видео (бесплатное), отсюда https://www.pexels.com/ru-ru/video/5390388/

В блоке Программа уроков смена иллюстрации при клике на соответственный пункт. 

![Lessons](https://user-images.githubusercontent.com/77831845/145990053-9fc02537-7a99-4c33-8a30-08b98da3fc0c.jpg)

![2](https://user-images.githubusercontent.com/77831845/145990169-4352793f-6eb5-4ae1-933d-89ab0fa8731c.png)
.

**Comment by MarinaDrumin** [comment]
фиолетовый темноват
курсивный розовый - есть ли варианты?
в остальном  мне понравилось))

**Reply by Darolga** [reply]
@MarinaDrumin 

_курсивный розовый - есть ли варианты?_

Другой шрифт курсивный подобрать или цвет розовый другой?

**Reply by MarinaDrumin** [reply]
> @MarinaDrumin
> 
> _курсивный розовый - есть ли варианты?_
> 
> Другой шрифт курсивный подобрать или цвет розовый другой?

  и цвет и шрифт

цвет хотелось бы более светлый и более в красный? 
про шрифт - мне нравится тема контраста с квадратным 

Upd - 
Цвет курсива попробуйте светло-голубой или мятный для фиолетового фона

**Comment by Darolga** [comment]
Хорошо!

**Comment by Darolga** [comment]
@MarinaDrumin 
Добрый вечер.
По цвету два новых варинта подготовила.
По рукописному шрифту - в варианте 3 наиболее удачный выбрала, в голубом или розовом цвете на выбор. 

![Lessons_vers_2](https://user-images.githubusercontent.com/77831845/146415148-aae5dc40-64fd-4eb9-a534-4b9ad8034646.jpg)


![Lessons_vers_3](https://user-images.githubusercontent.com/77831845/146415718-97cb4fb2-6cf7-4dc7-9396-f0f0bcc1ea2e.jpg)


![Lessons_vers_3_block_3](https://user-images.githubusercontent.com/77831845/146415853-853ee331-1c78-4614-9379-e7d8b91456a0.jpg)

**Comment by MarinaDrumin** [comment]
Оставляем первый фиолетовый вариант

**Comment by Darolga** [comment]
@MarinaDrumin 
@TorinAsakura 

Готов лендинг "Индивидуальные занятия".
Макет [здесь](https://www.figma.com/file/7FTyeashb3rns6sCdrsz3b/Landing?node-id=105%3A243) и ниже слайды.

Фоновое виде для первого экрана: https://www.pexels.com/ru-ru/video/5390388/

![Main](https://user-images.githubusercontent.com/77831845/146932675-759005fa-f58e-4e81-91d6-7e73b13a4338.jpg)

Попап-окно для заявки
![Pop up](https://user-images.githubusercontent.com/77831845/146932959-9b0beef6-cc49-4ef7-a5e4-211a0a25f32d.png)
![Pop up_2](https://user-images.githubusercontent.com/77831845/146932967-9f740cc1-3379-42f3-819e-8e569ebb6571.png)

**Reply by MarinaDrumin** [reply]
Слово пробный я не хотела бы использовать, так как подразумевается что пробный - бесплатный а у меня такого нет
варианты
- записаться на урок
- записаться на занятие
- записаться на консультацию

**Comment by Darolga** [comment]
@MarinaDrumin 
@TorinAsakura 

Слайды для "Программы уроков" (смена картинки при наведении на пункты текста)
![Programm_slide2](https://user-images.githubusercontent.com/77831845/146933204-dd551ebb-e491-4313-8efb-079c4e465eaf.png)
![Programm_slide3](https://user-images.githubusercontent.com/77831845/146933230-1b871771-aac4-4da9-a0ba-a8cdbc429c29.png)
![Programm_slide4](https://user-images.githubusercontent.com/77831845/146933243-16121450-efbb-45a0-b106-0c5d48a4d571.png)
![Programm_slide5](https://user-images.githubusercontent.com/77831845/146933261-07381ade-4741-4ab6-92e3-b90708da4505.png)
![Programm_slide6](https://user-images.githubusercontent.com/77831845/146933277-183368f5-7931-414e-8156-8999ea27f274.png)
![Programm_slide7](https://user-images.githubusercontent.com/77831845/146933295-4b3a9839-e884-4c4e-bb05-0ebabf6c0f74.png)

**Comment by Darolga** [comment]
@MarinaDrumin 
@TorinAsakura 

Слайды раздела "Вы получите", листаем по стрелкам
![You get_Slide2](https://user-images.githubusercontent.com/77831845/146933585-d12fe037-7673-48fb-a77f-859269d1ff47.png)
![You get_Slide3](https://user-images.githubusercontent.com/77831845/146933591-553bd1d3-48ed-4d8e-aeaa-442c6724e979.png)
![You get_Slide4](https://user-images.githubusercontent.com/77831845/146933597-19af0ea7-c1d4-4d00-93fd-597b0459f89d.png)
![You get_Slide5](https://user-images.githubusercontent.com/77831845/146933605-777c8918-71f0-4811-a45f-8da082a1dd50.png)

**Comment by Darolga** [comment]
@MarinaDrumin 
@TorinAsakura 

Слайды раздела "О преподавателе", открывается видео

Слайд 2 - [видео ](https://www.youtube.com/watch?v=RR6sSG2LoO4)
![About_teacher_slide_2](https://user-images.githubusercontent.com/77831845/146933844-1397ceea-c0c7-4154-9cfb-899d6d1aed1e.png)


Слайд 3 -[ видео ](https://www.youtube.com/watch?v=4k9fZs3ijx8)
![About_teacher_slide_3](https://user-images.githubusercontent.com/77831845/146934056-368e457a-90e5-4820-9e3b-10af28305ea9.png)

Слайд 4 - [видео ](https://www.youtube.com/watch?v=cI5dsy9nZVk)
![About_teacher_slide_4](https://user-images.githubusercontent.com/77831845/146934143-126f3219-d449-4972-9bd5-5217e98b3b65.png)

Слайд 5 - [видео ](https://www.youtube.com/watch?v=b5-1WS2QMdU)
![About_teacher_slide_5](https://user-images.githubusercontent.com/77831845/146934225-d2ea6759-0240-4351-9bfa-bc95d21a82d3.png)

Открытое видео
![Video_open](https://user-images.githubusercontent.com/77831845/146934277-52ba96c2-7145-49d0-92da-d566ba065eed.png)

**Reply by MarinaDrumin** [reply]
если вешать какое то одно видео  - я бы предпочла вот это https://youtu.be/wniVN7iEJqo

**Reply by MarinaDrumin** [reply]
Есть предложение в творческом пути использовать горизонтальную карусель

**Comment by Darolga** [comment]
@MarinaDrumin 
@TorinAsakura 

Слайды раздела "Отзывы учеников"
Видео открываются по аналогиис предыдущим разделом.

![Otziv_slide2](https://user-images.githubusercontent.com/77831845/146934457-31312d91-b01d-4826-b85e-1cbb97b57aef.png)
![Otziv_slide3](https://user-images.githubusercontent.com/77831845/146934468-9986331c-02fd-4500-a0c3-3455c1c9ce1e.png)

Ссылки на видеоотзывы:
Алексей Лазарев
https://youtu.be/Cqcf5nAoUQY
Лев Романов
https://youtu.be/NuvQJwLDezs
Светлана Бойкова 
https://youtu.be/DNxq1csn1hw
Александр Вершков 
https://youtu.be/gWXKpW8u-Wo
Даяна Эдгеева
https://youtu.be/ynU4OrlQOP8

**Reply by MarinaDrumin** [reply]
ок

**Comment by MarinaDrumin** [comment]
Отлично!) 
Продолжаем

**Comment by Darolga** [comment]
@MarinaDrumin 
Проверьте, пожалуйста, тексты

**Comment by MarinaDrumin** [comment]
По оформлению - все классно!
По тексту -  
материал взят со старого сайта где общий текст для двух сайтов

часть текста не актуальна для индивидуальных занятий

сделаю отдельный Гугл док с текстом

**Comment by MarinaDrumin** [comment]
https://docs.google.com/document/d/1I_Oc9Dlj41tcbY6wWK6IqV3HPwahkOZyDvuHTARqi5g/edit?usp=sharing

**Comment by MarinaDrumin** [comment]
для оперативной связи - ватсап или тг 89262464759
стараюсь максимально оперативно реагировать
во время репетиций и уроков не могу - приношу извинения за неудобства

**Comment by Darolga** [comment]
@MarinaDrumin 

Нет доступа к документу, дайте, пожйлуйста, darolga1@gmail.com
https://docs.google.com/document/d/1I_Oc9Dlj41tcbY6wWK6IqV3HPwahkOZyDvuHTARqi5g/edit?usp=sharing

**Reply by MarinaDrumin** [reply]
Попробуйте ещё раз

**Comment by MarinaDrumin** [comment]
Попробуйте ещё раз

**Comment by Darolga** [comment]
Есть

**Comment by Darolga** [comment]
@MarinaDrumin 

Марина, не будем делать блок "Вы научиесь, Вы получите", "Вы сможете" и т.д.?

![image](https://user-images.githubusercontent.com/77831845/147244204-f2706648-5637-4c70-8a0a-cba9b55bb023.png)

**Reply by MarinaDrumin** [reply]
Нет

**Reply by MarinaDrumin** [reply]
Ещё хочу дополнить текстом - постановка рук, координация и тд.

Завтра напишу.

**Reply by MarinaDrumin** [reply]
дополнила

**Comment by Darolga** [comment]
@MarinaDrumin 
Марина, добрый день, фото своего Диплома пришлите отдельным файлом, пожалуйста

**Reply by MarinaDrumin** [reply]
диплом https://disk.yandex.ru/i/QaxYv0GhJXyhZA

**Comment by MarinaDrumin** [comment]
ссылка на видео для сайта онлайн-курс
https://disk.yandex.ru/i/M0ti8b4dBij1Yg
